app/models/user.rb

```diff
@@ -6,6 +6,8 @@ class User < ApplicationRecord
 
   has_many :schedules, dependent: :destroy
 
+  has_and_belongs_to_many :groups # rubocop:disable Rails/HasAndBelongsToMany
```

Comment:
Rubocop wants you to use a `has_many :through` instead but this only makes sense when you want / care about attributes on the associative model, and we don't need those.